docs/TektonConfig.md

```diff
@@ -655,6 +668,14 @@ The following fields are supported in `HorizontalPodAutoscaler` (aka HPA)
 
 **NOTE**: If a Deployment or StatefulSet has a Horizontal Pod Autoscaling (HPA) and is in active state, Operator will not control the replicas to that resource. However if `status.desiredReplicas` and `spec.minReplicas` not present in HPA, operator takes the control. Also if HPA disabled, operator takes control. Even though the operator takes the control, the replicas value will be adjusted to the hpa's scaling range.
 
+#### webhookConfigurationOptions
+Defines additional options for each webhooks. Use webhook name as a key to define options for a webhook. Options are ignored if the webhook does not exist with the name key. To get detailed information about webhooks options visit https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
+
+the following options are supported for webhookConfigurationOptions
+* `failurePolicy` -  defines how unrecognized errors and timeout errors from the admission webhook are handled. Allowed values are `Ignore` or `Fail`
+* `timeoutSeconds` - allows configuring how long the API server should wait for a webhook to respond before treating the call as a failure.
+* `sideEffects` -  indicates whether the webhook have a side effet. Allowed values are `None`, `NoneOnDryRun`, `Unknown`, or 'Some'
```

Comment:
@jkhelil i see the valid values for `sideEffects` are `None` and `NoneOnDryRun` based on the official k8s doc https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#side-effects 

Can you please recheck for `Unknown`, or 'Some'

Reply:
I see all the 4 values here https://github.com/kubernetes/api/blob/master/admissionregistration/v1beta1/types.go#L74-L87 but not sure why k8s doc did not list this :thinking:

Reply:
I have picked the values from the code https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/admissionregistration/types.go#L122
and the values seems very old, so I am thinking the doc is not up to date